apmpackage/apm/0.1.0/manifest.yml

```diff
@@ -43,43 +43,36 @@ policy_templates:
         required: false
         show_user: true
         default: false
+      - name: api_key_limit
+        type: int
+        title: Maximum number of API Keys
+        description: Restrict number of unique API Keys per minute, used for auth between APM Agents and Server.
+        required: false
+        show_user: false
```

Comment:
nit: The `api_key_limit` is in the _Advanced Settings_ section (`show_user:false`). I find it easier to read to first group the keys that are always shown and then group the advanced settings together.

Reply:
ah, ok, hadn't noticed that. i was following the logic of putting related things together (so this bellow "enable api key")